admin/js/acc_user_importer-admin.js

```diff
@@ -266,6 +269,28 @@ var usersInData, roleRefreshed, newUsers, updatedUsers, usersWithErrors, accSync
 		});
 	}
 	
+	/**
+	 * Ask Wordpress to update database with parsed membership info.
+	 */
+	 function wpProccessExpiry () {
+
+		logLocalOutput("Now asking to process member expiry");
+		logLocalOutput("..");
+
+		var apiData = {'action': 'accUserAPI','request': 'processExpiry','security': ajax_object.nonce, 'dataset': JSON.stringify('')};
```

Comment:
Why are you using `JSON.stringify('')` here?

Reply:
I just cloned function wpProccessMembershipData() and modified it slightly to send a request to do membership expiry.  But I dont really have data to pass, so I pass a null string. I was not sure if it has to be in JSON format, so I left the JSON.stringify() as was done in wpProccessMembershipData().  I dont know too much what I am doing here, so if you have a suggestion, give me the replacement line of code and I will test it out.

Reply:
I think it would convert a JS object to a json string, but since we don't have any data, `'dataset': ''` or even omit the dataset key completely should work too.

Reply:
Thanks for the suggestion, I used 'dataset': '' and it works.